### PR TITLE
Add cross cutting consept part

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,21 @@ For full description, please refer to the official Berlin Group PSD2 specificati
 
 We have provided technical description in the [arc42 document](https://adorsys.github.io/xs2a-adapter/). 
 
+## Banks peculiarities
+
+There are some specific cases in communicating with banks that an XS2A Adapter user must be aware of.
```
Comment (andriimurashkin) [inline README.md:133]: communicating -> communication

File: docs/arc42/sections/8_cross-cutting_concepts.adoc [hunk 2]
```diff
@@ -1,15 +1,266 @@
...
+can pre- or post-handle a call to a bank.
+
+image::adapter-interceptor.png[XS2A Adapter Interceptor,width=100%]
+
+.XS2A Adapter Interceptor
+[cols=",",options="header",]
+|===
+|Class/Interface |Description
+|Interceptor |Interface provides the list of operations with default postHandle() implementation.
+|adorsys.AdorsysSigningHeadersInterceptor |Signs a request to banks that uses the Adorsys solutions.
```
Comment (HryhoriiHevorkian) [inline docs/arc42/sections/8_cross-cutting_concepts.adoc:173]: bankS -> that use

File: docs/arc42/sections/8_cross-cutting_concepts.adoc [hunk 3]
```diff
@@ -1,15 +1,266 @@
...
+
+In order to interact with the Banks' XS2A Interfaces, a user will need to have an eIDAS certificate. XS2A Adapter expects
+a user to supply a valid PKCS #12 Key Store file (.p12) with a certificate as a JVM parameter (standalone)
+or as a parameter in the Pkcs12KeyStore constructor (library).
+
+Description of the KeyStore and how to create it is already put down in https://github.com/adorsys/xs2a-adapter/blob/develop/docs/keystore.md[here]
+
+=== 8.8. XS2A Adapter Request Signing.
+
+XS2A Adapter can signing the request for a user, if an adapter applied `RequestSigningInterceptor` and has a QSEAL
```
Comment (HryhoriiHevorkian) [inline docs/arc42/sections/8_cross-cutting_concepts.adoc:249]: Can sign

**Review by HryhoriiHevorkian — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (berezkin88) [reply README.md]: amended

File: docs/arc42/sections/8_cross-cutting_concepts.adoc (hunk 2 above)
Comment (berezkin88) [reply docs/arc42/sections/8_cross-cutting_concepts.adoc]: amended

File: docs/arc42/sections/8_cross-cutting_concepts.adoc (hunk 3 above)
Comment (berezkin88) [reply docs/arc42/sections/8_cross-cutting_concepts.adoc]: amended

**Pull request merged**